zzz_generated/chip-tool/zap-generated/test/Commands.h

```diff
@@ -60546,7 +60546,7 @@ class TV_ApplicationLauncherClusterSuite : public TestCommand
         request.application.applicationId   = chip::Span<const char>("applicationIdgarbage: not in length on purpose", 13);
 
         request.data.Emplace();
-        request.data.Value() = chip::ByteSpan(chip::Uint8::from_const_char("datagarbage: not in length on purpose"), 4);
+        request.data.Value() = chip::ByteSpan(chip::Uint8::from_const_char("\x64\x61\x74\x61garbage: not in length on purpose"), 4);
```

Comment:
Is this ok? \x64 is actually a `d` so code seemed valid before. Generally all the items below seem to be more significantly changd to feel 'binary'. Why is that?

Reply:
I am updating this see comment below.

Reply:
Ok. Updated PR. Thank you.